.github/release.yml

```diff
@@ -0,0 +1,18 @@
+changelog:
+   categories:
+    - title: Features
+       labels:
+         - RN: MAJOR FEATURE
+         - RN: MINOR FEATURE
+    - title: Fixes
+       labels:
+         - RN: BUGFIX
+         - RN: SAFETY IMPROVEMENT
+         - RN: IMPROVEMENT
```

Comment:
I think improvement is not a fix, it should go in a separate  title.

Reply:
Yes that one can be a safety fix agree.

Reply:
> Yes that one can be a safety fix agree.

confirmed do you want a separate 'Improvements' category to host the 'RN: IMPROVEMENTS'?

Reply:
Yes major and minor

Reply:
Yes I meant Major and Minor Features also needs to included.

Reply:
> RN: Major and Minor Features also needs to be included.

the only RN label I've not got is: REFACTORING

what category would that be for?

Reply:
https://github.com/betaflight/betaflight/pulls?q=is%3Aopen+is%3Apr+label%3A%22RN%3A+REFACTORING%22

Reply:
Yes it's an different kind of improvement. No problem with this extra bullet point. Can we have the different RN label / tags being presented using collapsible sections containing markdown?

Reply:
> 

I'm not sure about that - this is what it says in the configuration guide:

"specify in YAML the pull request labels and authors you want to exclude from this release. You can also create new categories and list the pull request labels to be included in each of them."